### PR TITLE
Add possibility to disable audio output

### PR DESCRIPTION
**Pull request by dima-xd**
Might fix some games (e.g The Legend of Zelda: Skyward Sword HD)

### CODE REVIEW

File: app/src/main/cpp/skyline/audio.cpp [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 namespace skyline::audio {
     Audio::Audio(const DeviceState &state) : oboe::AudioStreamCallback() {
+        settings = state.settings;
```
Comment (bylaws) [inline app/src/main/cpp/skyline/audio.cpp:8]: Use {} initialiser list

File: app/src/main/cpp/skyline/services/audio/IAudioRenderer/IAudioRenderer.cpp [hunk 2]
```diff
@@ -68,7 +69,9 @@ namespace skyline::service::audio::IAudioRenderer {
         for (u32 i{}; i < effectsIn.size(); i++)
             effects[i].ProcessInput(effectsIn[i]);
 
-        UpdateAudio();
+        if (!*state.settings->isAudioOutputDisabled) {
```
Comment (bylaws) [inline app/src/main/cpp/skyline/services/audio/IAudioRenderer/IAudioRenderer.cpp:72]: Drop {}

File: app/src/main/java/emu/skyline/utils/NativeSettings.kt [hunk 3]
```diff
@@ -28,6 +28,9 @@ class NativeSettings(context : Context, pref : PreferenceSettings) {
     var executorSlotCount : Int = pref.executorSlotCount
     var enableTextureReadbackHack : Boolean = pref.enableTextureReadbackHack
 
+    //Audio
```
Comment (bylaws) [inline app/src/main/java/emu/skyline/utils/NativeSettings.kt:31]: Add space after //

File: app/src/main/java/emu/skyline/utils/PreferenceSettings.kt [hunk 4]
```diff
@@ -41,6 +41,9 @@ class PreferenceSettings @Inject constructor(@ApplicationContext private val con
     var executorSlotCount by sharedPreferences(context, 6)
     var enableTextureReadbackHack by sharedPreferences(context, false)
 
+    //Audio
```
Comment (bylaws) [inline app/src/main/java/emu/skyline/utils/PreferenceSettings.kt:44]: Add space after //

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -75,6 +75,11 @@
     <string name="enable_texture_readback_hack">Enable Texture Readback Hack</string>
     <string name="enable_texture_readback_hack_enabled">Texture readback hack is enabled (Will break some games but others will have higher performance)</string>
     <string name="enable_texture_readback_hack_disabled">Texture readback hack is disabled (Ensures highest accuracy)</string>
+    <!-- Settings - Audio -->
+    <string name="audio">Audio</string>
+    <string name="disable_audio_output">Disable Audio Output</string>
+    <string name="disable_audio_output_enabled">Audio output is disabled (Might fix some games with audio issues)</string>
```
Comment (bylaws) [inline app/src/main/res/values/strings.xml:81]: Would drop the might fix...

**Review by bylaws — CHANGES_REQUESTED** [review]

**Review by lynxnb — APPROVED** [review]

**Pull request merged**